src/modm/ui/color/hsv.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2009, Martin Rosekeit
+ * Copyright (c) 2009-2013, Fabian Greif
+ * Copyright (c) 2012-2013, 2015, Niklas Hauser
+ * Copyright (c) 2013, David Hebbeker
+ * Copyright (c) 2021, Thomas Sommer
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#ifndef MODM_COLOR_HSV_HPP
+#define MODM_COLOR_HSV_HPP
+
+#include <stdint.h>
+
+#include <modm/io/iostream.hpp>
+
+#include "rgb.hpp"
+#include "rgb565.hpp"
+
+namespace modm
+{
+namespace color
+{
+
+template<typename UnderlyingType>
+class RgbT;
+
+class Rgb565;
+/// @ingroup modm_ui_color
+template<typename UnderlyingType = uint8_t>
```

Comment:
Do we want to use `T` instead of `UnderlyingType`? It would be more consistent with the rest of modm.